src/routes/api-documentation.ts

```diff
@@ -35,6 +37,52 @@ export default (router: Router): Middleware => (
             },
           ],
         },
+        {
```

Comment:
so it's the https://www.w3.org/ns/hydra/core#apiDocumentation that would be used by a generic client to build a form? Rather than embedded controls in an API response like with https://schema.org/Action?

Reply:
It'll be a mix; a Hydra client merges them. API details that apply everywhere go into the documentation, resource-specific details go into the resource itself (eg can update any article vs can update specific articles).

Related: https://httpapis.slack.com/archives/C1JP575EX/p1572601301095800